ramp-engine/ramp_engine/aws/api.py

```diff
@@ -144,6 +143,7 @@ def launch_ec2_instances(config, nb=1):
     sess = _get_boto_session(config)
     client = sess.client('ec2')
     resource = sess.resource('ec2')
+    on_demand = False
```

Comment:
To make this clearer, I think we should change this var name to `switch_to_on_demand`